albumentations/core/composition.py

```diff
@@ -379,16 +387,25 @@ class BboxParams(Params):
             visible area in pixels is less than this value will be removed. Default: 0.0.
         min_visibility (float): minimum fraction of area for a bounding box
             to remain this box in list. Default: 0.0.
+        check_each_transform (bool): if `True`, then bboxes will be checked after each dual transform.
+            Default: `True`
```

Comment:
The docstring says that the default value is `True`, but the default value in `__init__` is set to `False`.

Reply:
Changed default value to True